src/equations/compressible_euler_quasi_1d.jl

```diff
@@ -0,0 +1,371 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    CompressibleEulerEquationsQuasi1D(gamma)
+
+The quasi-1d compressible Euler equations
+```math
+\frac{\partial}{\partial t}
+\begin{pmatrix}
+a \rho \\ a \rho v_1 \\ a e
+\end{pmatrix}
++
+\frac{\partial}{\partial x}
+\begin{pmatrix}
+a \rho v_1 \\ a \rho v_1^2 \\ a v_1 (e +p)
+\end{pmatrix}
++ 
+a \frac{\partial}{\partial x}
+\begin{pmatrix}
+0 \\ p \\ 0    
+\end{pmatrix}
+=
+\begin{pmatrix}
+0 \\ 0 \\ 0
+\end{pmatrix}
+```
+for an ideal gas with ratio of specific heats `gamma` in one space dimension.
+Here, ``\rho`` is the density, ``v_1`` the velocity, ``e`` the specific total energy **rather than** specific internal energy, 
+``a`` the (possibly) variable nozzle width, and
+```math
+p = (\gamma - 1) \left( e - \frac{1}{2} \rho v_1^2 \right)
+```
+the pressure.
+
+The nozzle width function ``a(x)`` is set inside the initial condition routine
```

Comment:
I am not sure if $a(x)$ is the ideal choice for the nozzle width, as it might be confused with e.g. a space-varying advection velocity.

Reply:
At least in a few of the papers we read, `a` was the variable used to represent channel width. Also, it is currently used to represent channel width for the quasi 1D shallow water equations, so I think we should keep it the same until we merge this. Then, if we want to change, we can change both instances at the same time.

Reply:
Ah okay, I was not aware that $a$ is also used for quasi 1D shallow water.